UltimateMedals.as

```diff
@@ -500,35 +515,35 @@ void OnSettingsChanged() {
 	UpdateText();

```

Comment:
Closest line i could write this but in Update() you should add support for renaming the champion and warrior medals like the other medals.